source/android/mobile/src/main/res/drawable/layered_edit_text_styles.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android" xmlns:adaptivecards="http://schemas.android.com/apk/res-auto" >
+    <item adaptivecards:adaptive_state_input_invalid="true" android:drawable="@drawable/layered_edit_text_invalid" />
```

Comment:
Consider using `adaptivecards:state_error`. We can use it in the future for non-inputs that can be in an error state (e.g. an Action, an entire AdaptiveCard).

There is no need to prefix `adaptive_` because it is already restricted to our namespace.

Android's existing states are in the following formats: "state_selected", "state_pressed", etc..

Reply:
changed the name to use state_error